apps/server/src/modules/deletion/controller/dto/deletion-request.body.params.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNumber } from 'class-validator';
+import { EntityId } from '@shared/domain';
+import { DeletionDomainModel } from '../../domain/types/deletion-domain-model.enum';
```

Comment:
Please add proper "barrel" files (`index.ts` exporting this enum) to not do the direct file import.